packages/paste-website/src/components/homepage/GetStarted.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/core/box';
+import {Grid, Column} from '@twilio-paste/core/grid';
+
+import {GetStarterWhy} from './GetStartedWhy';
+import {GetStartedInclusive} from './GetStartedInclusive';
+import {GetStartedRunning} from './GetStartedRunning';
+
+export const GetStarted: React.FC = () => {
+  return (
+    <Box as="section">
```

Comment:
Is it be possible to remove this Box and add the `as` to Grid directly?

Reply:
Hopefully, I guess. Grid should support it